label_studio/io_storages/s3/api.py

```diff
@@ -38,7 +80,13 @@
 @method_decorator(
     name='post',
     decorator=swagger_auto_schema(
-        tags=['Storage:S3'], operation_summary='Create new storage', operation_description='Get new S3 import storage'
+        tags=['Storage:S3'],
+        x_fern_sdk_group_name=['import_storage', 's3'],
+        x_fern_sdk_method_name='create',
+        x_fern_audiences=['public'],
+        operation_summary='Create new S3 storage',
+        operation_description='Get new S3 import storage',
```

Comment:
These seem a little contradictory, though it looks like that problem existed before this PR